webapp/src/app/match-list/match-list.component.ts

```diff
@@ -47,7 +47,10 @@ export class MatchListComponent implements OnChanges {
         let lastMatchDate = '';
         for (let i = 0; i < this.cols; i++) {
             const matchCol = [];
-            for (let j = 0; (j < this.minItemsPerColumn || j < numberOfMatchesPerColumn) && matchIndex < this.matches.length; j++) {
+            for (let j = 0; (j < this.minItemsPerColumn
+                || j < numberOfMatchesPerColumn)
+                && matchIndex < this.matches.length; j++) {
```

Comment:
Being totally honest here... this still looks messy. Even with this change.

Reply:
I tried fixing this on my lint-fixes branch, but this logic seems so weird, I just gave up. I believe this will be dropped on future refactors, so I'm okay with leaving this as it is for now.